/ZLAC:
  ros__parameters:
    AW: 0.445
    WD: 0.23
    WO1: 0.391
    WO2: 0.391
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false

/dynamixel_controller:
  ros__parameters:
    AW: 0.445
    WD: 0.23
    WO1: 0.391
    WO2: 0.391
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false

/velocity_smoother:
  ros__parameters:
    smoothing_frequency: 20.0
    max_velocity: [3.0, 0.0, 2.0]
    min_velocity: [-3.0, 0.0, -2.0]
    deadband_velocity: [0.0, 0.0, 0.0]
    velocity_timeout: 1.0
    max_accel: [1.0, 0.0, 1.0]
    max_decel: [-1.0, 0.0, -1.0]
